pageObject/loginPage.js

```diff
@@ -1,10 +1,12 @@
 let Input = require('../elements/input');
 let Button = require('../elements/button');
 let BasePage = require('../base/bagePage');
+let TextBox = require('../elements/textBox');
 
 let emailInputLocator = by.xpath('//*[@id="email"]');
 let passwordInputLocator = by.xpath('//*[@id="passwd"]');
 let submitButton = by.xpath('//*[@id="SubmitLogin"]')
+let errorMessageLocator = by.css('ol > li');
```

Comment:
Please try to improve locator, use some class name or ID. Because each html ordered list have this structure `ol > li`